exercises/strings/longestWord/longestWord.js

```diff
@@ -9,11 +9,26 @@
 
 function longestWord(string) {
   // This is your job. :)
+  let words = []
+  let endIndex = 0
+  let startIndex = 0
+  for (i in string) {
```

Comment:
Don't use `for ... in`. If you want to iterate over elements / characters directly, use `for ... of`. If you want to iterate over indexes, use `for(let ...; ...; ...)`.  `for ... in` is a mistake in the JS language and doesn't do what you expect in every context. It _will_ come to bite you.